docs/source/examples.rst

```diff
@@ -8,7 +8,8 @@ You can download the source for these notebooks `here <https://github.com/ioos/i
 
 * :doc:`Read water level data from CSV into Pandas, run QARTOD tests, and plot results in Bokeh </examples/QartodTestExample_WaterLevel>`
 
-* :doc:`Run QARTOD on a netCDF file, and store the results </examples/Qartod_netCDF_example>`
+..
+    * :doc:`Run QARTOD on a netCDF file, and store the results </examples/Qartod_netCDF_example>`
```

Comment:
We need to re-run all these notebooks and make sure they are updated. I'm skipping them for now b/c I could not make them work.